hi-ml/testhiml/testhiml/test_attentionlayers.py

```diff
@@ -18,8 +18,11 @@ def _test_attention_layer(attentionlayer: nn.Module, dim_in: int, dim_att: int,
     row_sums = sum(attn_weights, dim=1, keepdim=True)
     assert allclose(row_sums, ones_like(row_sums))
 
-    pooled_features = attn_weights @ features.flatten(start_dim=1)
-    assert allclose(pooled_features, output_features)
+    if isinstance(attentionlayer, TransformerPooling):
+        pass
```

Comment:
Does the trick of adjusting for the CLS token attention not make the Transformer attentions sum to 1?